addons/project_issue/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,70 @@
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+project_issue / project.issue            / progress (float)              : module is now 'project_issue_sheet' ('project_issue')
+# Nothing to do
+
+project_issue / project.issue            / section_id (many2one)         : relation is now 'crm.team' ('crm.case.section')
+project_issue / project.issue            / section_id (many2one)         : was renamed to team_id [nothing to to]
+# Nothing to do
+
+project_issue / project.issue            / version_id (many2one)         : DEL relation: project.issue.version
+# Feature removed ---> Nothing to do
+
+project_issue / project.issue            / website_message_ids (one2many): NEW relation: mail.message
+# Field doesn't exist neither in v8 or in v9 (probably related to Mail module) ---> Nothing to do
+
+project_issue / project.project          / label_issues (char)           : NEW
+# Nothing to do
+
+project_issue / project.project          / project_escalation_id (many2one): DEL relation: project.project
+# Nothing to do
+
+---XML records in module 'project_issue'---
+
+NEW ir.actions.act_window: project_issue.action_project_issue_report_filtered
+NEW ir.actions.act_window: project_issue.open_view_issue
+DEL ir.actions.act_window: project_issue.project_issue_version_action
+DEL ir.actions.act_window.view: project_issue.action_project_issue_report_graph
+DEL ir.actions.todo: base.open_menu
+# ir.actions.* ---> Nothing to do
+
+NEW ir.model.access: project_issue.access_crm_team
+NEW ir.model.access: project_issue.access_project_issue_portal
+DEL ir.model.access: project_issue.access_crm_case_section
+DEL ir.model.access: project_issue.access_project_issue_version_project
+DEL ir.model.access: project_issue.access_project_issue_version_project_user
+# ir.model.access ---> Nothing to do.
+
+NEW ir.rule: project_issue.portal_issue_rule
+# NEW ir.rule --> Nothing to do
+
+NEW ir.ui.menu: project_issue.portal_services_issues
+DEL ir.ui.menu: project_issue.menu_project_confi
+DEL ir.ui.menu: project_issue.menu_project_issue_version_act
+NEW ir.ui.view: project_issue.project_issue_pivot_view
+NEW ir.ui.view: project_issue.view_project_issue_report_pivot
+DEL ir.ui.view: project_issue.project_issue_version_form_view
+DEL ir.ui.view: project_issue.project_issue_version_search_view
+DEL ir.ui.view: project_issue.view_config_settings
+# ir.ui.* ---> Nothing to do
+
+NEW mail.message.subtype: project_issue.mt_project_issue_ready
+DEL mail.message.subtype: project_issue.mt_issue_assigned
+DEL mail.message.subtype: project_issue.mt_project_issue_assigned
+# Related to mail module changes ---> Nothing to do
+
+DEL project.issue.version: project_issue.type1
+DEL project.issue.version: project_issue.type2
+DEL project.issue.version: project_issue.type3
+# Feature removed ---> Nothing to do
```

Comment:
If we preserve version_id, we should preserve this also, but I'm not sure if at the end it deserves to be preserved or is a dead feature.